src/hooks/useVenuePoll.ts

```diff
@@ -19,10 +19,10 @@ export const useVenuePoll = () => {
   const { userId } = useUser();
 
   const voteInPoll = useCallback(
-    (pollVote: PollVoteBase) => {
+    async (pollVote: PollVoteBase) => {
       if (!venueId) return;
 
-      voteInVenuePoll({ pollVote, venueId });
+      await voteInVenuePoll({ pollVote, venueId });
```

Comment:
just return the value

Reply:
fixed in 1d3b5688b0f98e34f710a6d2cb8fb2ca296afcb7